testing/xmrig/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Mainteiner: Oleg Titov <oleg.titov@gmail.com>
+
+pkgname=xmrig
+pkgver=2.8.3
+pkgrel=0
+pkgdesc="XMRig is a high performance Monero (XMR) miner"
+url="https://github.com/xmrig/xmrig"
+arch="x86_64 x86 aarch64 armhf armv7"
+license="GPL-3.0"
+depends="libuv libmicrohttpd openssl"
+makedepends="cmake libmicrohttpd-dev libuv-dev openssl-dev"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/xmrig/xmrig/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	default_prepare
+}
+
+build() {
+	cd "$builddir"
+	mkdir build
+	cd build
+	cmake ..
```

Comment:
I bet this needs option to link libuv statically https://github.com/xmrig/xmrig-proxy/wiki/Ubuntu-Build#additional-cmake-options

Reply:
Works quite fine with dynamic linking. I do not know under which circumstances we might need static linking for libuv.